bucket/ngspice.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "31",
+    "description": "SPICE simulator for electric and electronic circuits",
+    "homepage": "http://ngspice.sourceforge.net",
+    "license": "GFDL-1.3-or-later",
```

Comment:
Where did you get this information?

>  The ngspice license is the `Modified BSD' license. This is adopted for all of its source code, test and example files except for the files listed below.

>-------------------------------------- ngspice ---------------------------------
---------------------------------- 'Modified BSD' ------------------------------
>
> Copyright 1985 - 2018, Regents of the University of California and others
Redistribution and use in source and binary forms, with or without modification,
are permitted provided that the following conditions are met:
> 
> 1. Redistributions of source code must retain the above copyright notice,
this list of conditions and the following disclaimer.
> 2. Redistributions in binary form must reproduce the above copyright notice,
this list of conditions and the following disclaimer in the documentation
and/or other materials provided with the distribution.
> 3. Neither the name of the copyright holder nor the names of its
contributors may be used to endorse or promote products derived from this
software without specific prior written permission.
> 
> THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
POSSIBILITY OF SUCH DAMAGE.

docs/COPYING

Reply:
Thanks for the details.  

The homepage has this line in the bottom.
> All text is available under the terms of the GNU Free Documentation License
apparently this is referring to the page contents, but not *Ngspice* itself.

I cannot find the full text of the "modified BSD" under *Ngspice*'s website. Therefore I just mark the `license` as "Freeware", and add a comment that mentions the file.

Reply:
[Here](https://sourceforge.net/p/ngspice/ngspice/ci/master/tree/COPYING) is the license.  It's the same as [BSD-3-Clause](https://spdx.org/licenses/BSD-3-Clause.html)

Reply:
Oops, forgot that I can just look into the code repository. Thanks for the info!